src/components/index.js

```diff
@@ -4,7 +4,8 @@ export { default as CourseInfo } from './course/CourseInfo';
 
 // ========================================
 // lesson
-export { default as LessonInfo } from './lesson/LessonInfo';
+// export { default as LessonPage } from './lesson/LessonInfo';
```

Comment:
You should still import all of your new components here

```suggestion
export { default as LessonInfo } from './lesson/LessonInfo';
export { default as LessonVideo } from './lesson/LessonVideo';
export { default as LessonsPane } from './lesson/LessonsPane';
```